src/app/cli/src/app.rs

```diff
@@ -467,6 +470,7 @@ pub fn configure_base_catalog(
     b.add::<CreateDatasetFromSnapshotUseCaseImpl>();
     b.add::<DeleteDatasetUseCaseImpl>();
     b.add::<RenameDatasetUseCaseImpl>();
+    b.add::<PushStatusServiceImpl>();
```

Comment:
Place the registration near `PushService`. This is a list of use cases. No behavioral effect, but there is at least some order in this registraiton code.